src/ui/ui-kit/FormFieldset/FormFieldset.tsx

```diff
@@ -66,12 +66,16 @@ export const FormFieldset = React.forwardRef<
         }}
       >
         <VStack gap={4} style={{ width: '100%' }}>
-          <HStack gap={8} justifyContent="space-between" alignItems="center">
-            <UIText kind="small/regular" as="label" ref={labelRef}>
-              {title}
-            </UIText>
-            {endTitle ? <UIText kind="small/regular">{endTitle}</UIText> : null}
-          </HStack>
+          {title || endTitle ? (
+            <HStack gap={8} justifyContent="space-between" alignItems="center">
+              <UIText kind="small/regular" as="label" ref={labelRef}>
+                {title}
+              </UIText>
```

Comment:
in case of falsy `title` well have empty node here, I assume this is on purpose, right? So that the `endTitle` stays on end?

Reply:
yes that was the intention